service/templates/task_perf_data.html

```diff
@@ -360,7 +360,7 @@ <h5>Add comparison to</h5>
       <div class="muted" ng-show="!locked">click on a graph to lock to a commit</div>
       <div class="perf-trendchart-wrapper" ng-repeat="k in perfSample.testNames()">
         <a name="[[k]]"></a>
-        <h4> <i class="fa" ng-class="isGraphHidden(k) ? 'fa-caret-right' : 'fa-caret-down'" ng-click="toggleGraph(k)"></i>&nbsp;[[k]] <i class="fa fa-sticky-note" ng-show="perfSample.resultForTest(k).isExpandedMetric" title="Expanded metric"></i></h4>
+        <h4> <i class="fa" ng-class="isGraphHidden(k) ? 'fa-caret-right' : 'fa-caret-down'" ng-click="toggleGraph(k)"></i>&nbsp;[[k]]<span ng-show="isCanary(k)"> - Canary Test </span><i class="fa fa-sticky-note" ng-show="perfSample.resultForTest(k).isExpandedMetric" title="Expanded metric"></i></h4>
```

Comment:
Is this needed? It might confuse people who think it's a part of the test name

Reply:
The ticket mentions a visual display marking canary tests in addition to folding them down by default. I've used this since I couldn't find any bird-like icons in font-awesome. I'm open to anything though -- do we have a set of assets we draw from in addition to font awesome; is licensing a concern for our internal tools?

Reply:
Font-awesome is free and can be used in any sort of project without licensing concerns. If you'd like to add other icon libraries you can feel free to do so, however I don't think it's too important to have it specifically be a bird. The main thing is to leave the header as just the test name

Reply:
maybe **fa-balance-scale**, since the canary tests are trying to ensure that the nodes have the even performance.

A hover on the icon stating that this represents a canary would also be useful.